containers/base/Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
 
 # Setup Node.js
```

Comment:
Probably good to clearly indicate we are using the LTS build (rather than cutting edge/latest).